public_suffix_list.dat

```diff
@@ -6584,6 +6584,17 @@ lib.wy.us
 pvt.k12.ma.us
 chtr.k12.ma.us
 paroch.k12.ma.us
+// Merit Network, Inc. maintains the registry for =~ /(k12|cc|lib).mi.us/ and the following
+//    see also: http://domreg.merit.edu
+//    see also: whois -h whois.domreg.merit.edu help
```

Comment:
Unfortunately, I can't use the web-editor to suggest changes and feedback, but we've been trying to tighten up the internal format/documentation for these additions.

Since all of this discussion is available via history, would you be able to put something simpler:

// Regional and industry-related 3LDs offered by the mi.us registrar - see http://domreg.merit.edu
// Submitted by mi.us registry <email here>

@gerv @weppos - Should we be putting these under ICANN or PRIVATE? I'm not sure how we want to handle that distinction - is it the same as a `.com.ccTLD` case operated by the `.ccTLD` registry, or do we see it more like `appspot.com` - the party who is delegated `appspot.com` (e.g. Google) is then sub-delegating? I can see the argument either way, and I'll note it somewhat touches on what Peter Bowen raised recently.